packages/openapi-fetch/src/index.ts

```diff
@@ -122,9 +122,11 @@ export default function createClient<Paths extends {}>(clientOptions: ClientOpti
       redirect: "follow",
       ...options,
       ...init,
-      headers: baseHeaders,
```

Comment:
I don’t think this line needs to be deleted; I believe the reference is the same either way so the `baseHeaders.delete` will still work, no?

Reply:
Yep! I've undone that and confirmed it works as intended.